office-365-management-api/office-365-management-activity-api-schema.md

```diff
@@ -72,6 +72,7 @@ This article provides details on the Common schema as well as service-specific s
 |[Cloud Update profile configuration schema](#cloud-update-profile-configuration-schema)| Extends the Common schema with the properties specific to the Cloud Update profile configuration audit data.|
 |[Cloud Update tenant configuration schema](#cloud-update-tenant-configuration-schema)| Extends the Common schema with the properties specific to the Cloud Update tenant configuration audit data.|
 |[Cloud Update device configuration schema](#cloud-update-device-schema)| Extends the Common schema with the properties specific to the Cloud Update device configuration audit data.|
+|[AAD Risk Detection schema](#aad-risk-detection-schema)| Extends the Common schema with the properties specific to AAD Risk Detection events.|

```

Comment:
I thought "AAD" was retired and we now use "Entra". Can you verify it is OK to use in this context?